kubetest/kops.go

```diff
@@ -273,17 +309,19 @@ func (k kops) Up() error {
 		"--node-count", strconv.Itoa(k.nodes),
 		"--node-volume-size", strconv.Itoa(k.diskSize),
 		"--master-volume-size", strconv.Itoa(k.diskSize),
+		// TODO chrislovecnm make this configurable
+		"--master-size", "c4.large",
```

Comment:
We override this in some tests (the ENA test IIRC), so this should be a flag if you want to default it to c4.large